projects/admin/src/app/menu/menu.component.ts

```diff
@@ -128,7 +130,7 @@ export class MenuComponent implements OnInit {
     this.recordTypes = [{
       type: 'documents',
       field: 'autocomplete_title',
-      getSuggestions: (query, persons) => this.getDocumentsSuggestions(query, persons)
+      getSuggestions: (query, documents) => this.getDocumentsSuggestions(query, documents)
```

Comment:
Was it a bug or a change ?

Reply:
I think it's cosmetic, but I find illogical to send a variable `persons` when querying **document** suggestions